packages/aws-cdk-lib/core/lib/tag-manager.ts

```diff
@@ -303,7 +303,7 @@ export class TagManager {
    */
   public static isTaggable(construct: any): construct is ITaggable {
     const tags = (construct as any).tags;
-    return tags && typeof tags === 'object' && (tags as any)[TAG_MANAGER_SYM];
+    return typeof tags === 'object' && (tags as any)[TAG_MANAGER_SYM];
```

Comment:
But this now explodes if `tags` is `null`. It shouldn't be, but it could be.

Sillily, in JS it's:

```
typeof null == 'object'
```

So you can't really get away from the truthiness test. The only thing you have to do is make sure the truth-tested falsy value doesn't leak through.

Reply:
Good to know - just updated the return statement:

```ts
return tags !== undefined && tags !== null && ...
```